src/test/java/Homework18.java

```diff
@@ -0,0 +1,42 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class Homework18 extends BaseTest {
+
+
+    @Test
+    public void playSong() throws InterruptedException {
+        //login
+        provideEmail("demo@class.com");
+        providePassword();
+        clickSubmitBtn();
+
+        playASong();
+        validateSongIsPlaying();
+
+    }
+    private void playASong() throws InterruptedException {
+
+        WebElement nextBtn = driver.findElement(By.xpath("//i[@data-testid=\"play-next-btn\"]"));
+        nextBtn.click();
+
+        Thread.sleep(2000);
```

Comment:
you can use instead sleep wait 
example wait.until(ExpectedConditions.invisibilityOfElementLocated(YouElement);